composer.json

```diff
@@ -16,17 +16,17 @@
         }
     ],
     "require": {
-        "php": "^7.0",
-        "illuminate/support": "^5.4.30",
+        "php": "^7.1",
+        "illuminate/support": "^6.0",
```

Comment:
Can't we allow 5.4 and up here as well?

Reply:
L5.4 is out of support since *January 24th, 2018*. In addition to https://github.com/spatie/laravel-medialibrary/pull/1542#discussion_r318140512